src/Containers/App/index.js

```diff
@@ -3,6 +3,7 @@ import React, { Suspense, lazy } from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import './style.css';
 import BlobField from 'src/BlobField';
+import Info from 'src/Containers/Info';
```

Comment:
not sure if this should go in `src/Containers`, `src/Components`, or above that on the same level as `BlobField`

Reply:
I think containers. Generally those are pretty much equivalent to "pages" BlobField should probably go in components if we wanted to be very buttoned up.